AsyncDisplayKit/ASCollectionView.mm

```diff
@@ -202,6 +204,12 @@ - (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionVi
   // and should not trigger a relayout.
   _ignoreMaxSizeChange = CGSizeEqualToSize(_maxSizeForNodesConstrainedSize, CGSizeZero);
   
+  // Register the default layout inspector delegate for flow layouts, custom layouts
+  // will need to roll their own ASCollectionViewLayoutInspecting implementation.
+  _layoutDelegate = [self flowLayoutInspector];
```

Comment:
Yeah, I think we should refactor this to be = [self layoutInspector] (rename the method) at the least, even if it only ever returns nil or a flow layout inspector.  The comment should also be slightly tweaked to indicate this is the behavior, just making clear that it only has a default for flow layouts.